### PR TITLE
[delete #97] Soft delete data package

### PR DESCRIPTION
**Pull request by subhankarb**

### CODE REVIEW

File: app/mod_api/controllers.py [hunk 1]
```diff
@@ -85,6 +85,23 @@ def save_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/package/<publisher>/<package>", methods=["DELETE"])
+def delete_data_package(publisher, package):
+    try:
```
Comment (zelima) [inline app/mod_api/controllers.py]: This is missing Doc-string for Flasgger. So I  am not able to manually test @subhankarb could you please fix this?

**Comment by zelima** [comment]
@subhankarb Also for future, please include code tidy ups in separate commit rather in same that actual changes are. Eg Zappa deploy and Travis thing

**Comment by subhankarb** [comment]
Thanks. I will do it

File: app/mod_api/controllers.py (hunk 1 above)
Comment (subhankarb) [inline app/mod_api/controllers.py]: Done.

**Pull request merged**